crates/router/src/connector/noon/transformers.rs

```diff
@@ -660,6 +662,7 @@ impl From<NoonWebhookObject> for NoonPaymentsResponse {
                     //For successful payments Noon Always populates error_code as 0.
                     error_code: 0,
                     error_message: None,
+                    reference: value.reference,
```

Comment:
Hi @sudharsangs ,

As per api-documentation of [Noon](https://docs.noonpayments.com/webhook) , they don't send any merchant reference field in Webhook response . They only send merchant reference id incase of `allowedRetry ` . Could you please make it as None ,since it's a optional field .

Reply:
Then I can even remove reference from `NoonWebhookObject` struct right?

Reply:
Yes

Reply:
Done